src/react/boosts/Boost.tsx

```diff
@@ -11,13 +11,15 @@ export interface BoostProps {
 
 export interface BoostStates {
   payment: boolean;
+  newsroom: string;
```

Comment:
Minor, but we often use `newsroom` to refer to the newsroom object or data wrapper, and as a string it could be a couple things, I'd specify whether it's newsroom name or address or something else.